kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/marshaller/included/DMNMarshallerImportsClientHelper.java

```diff
@@ -199,10 +218,14 @@ public void onSuccess(final Definitions definitions) {
 
                                     @Override
                                     public void onError(final ClientRuntimeError error) {
+                                        LOGGER.warning(error::getMessage);
```

Comment:
@jomarko Because I was not totally sure that flow should be considered an error. If you read the comment below line 221: `//Swallow. Since it must try to load other paths.`, it seems to be this case is not managed as an error flow. Moreover, at line 223, the promise is resolved as a success. I didn't investigate the reason for this behavior, so I decided to add a log for this case considering it as a Warning that an error.

Reply:
Probably this https://github.com/kiegroup/kogito-editors-java/blob/76ce46c187a04f5f33d793fb7de3e43b8c05b262/kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/marshaller/included/DMNMarshallerImportsClientHelper.java#L197 (if any promise is rejected `Promise.all()` will fail too).